azure/package.json

```diff
@@ -31,7 +31,7 @@
     "clean": "lerna run clean --stream --parallel && npm run clean:docs && npm run clean:nyc",
     "clean:docs": "rimraf **/_api-extractor-temp docs/api/*/**",
     "clean:nyc": "rimraf nyc/**",
-    "format": "lerna run format --no-sort --stream -- -- -- --color",
```

Comment:
"color" is not a prettier cli option as far as I can tell. I'm also of the opinion that we shouldn't be passing arguments down this way, as it makes (IMO) creepy assumptions about script structure.